ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -176,7 +173,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 13;
 			files = (
-				C4AE09092AC43A4800E13758 /* OneSignalNotificationServiceExtension.appex in Embed Foundation Extensions */,
 			);
 			name = "Embed Foundation Extensions";
 			runOnlyForDeploymentPostprocessing = 0;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems like the `OneSignalNotificationServiceExtension.appex` has been removed from the "Embed Foundation Extensions" build phase. If this extension is no longer needed, then it's fine. However, if it was removed unintentionally, you might want to add it back.

```diff
 			files = (
+				C4AE09092AC43A4800E13758 /* OneSignalNotificationServiceExtension.appex in Embed Foundation Extensions */,
 			);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->